daprdocs/content/en/operations/hosting/self-hosted/self-hosted-overview.md

```diff
@@ -12,7 +12,7 @@ Dapr can be configured to run in self-hosted mode on your local developer machin
 
 ## Initialization
 
-Dapr can be initialized [with Docker]({{< ref self-hosted-with-docker.md >}}) (default) or in [slim-init mode]({{< ref self-hosted-no-docker.md >}}). The default Docker setup provides out of the box functionality with the following containers and configuration:
+Dapr can be initialized [with Docker]({{< ref self-hosted-with-docker.md >}}) (default) or in [slim-init mode]({{< ref self-hosted-no-docker.md >}}). It can run in [airgap environemnt]({{< ref self-hosted-airgap.md >}}) as well. The default Docker setup provides out of the box functionality with the following containers and configuration:
```

Comment:
```suggestion
Dapr can be initialized [with Docker]({{< ref self-hosted-with-docker.md >}}) (default) or in [slim-init mode]({{< ref self-hosted-no-docker.md >}}). It can also be initialized and run in [offline or airgap environments]({{< ref self-hosted-airgap.md >}}). The default Docker setup provides out of the box functionality with the following containers and configuration:
```